### ISSUE TITLE
NTR: Eosinophilic cystitis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3953]
New term request: Eosinophilic cystitis. Synonyms: 
Definition: Characterized by transmural inflammation of the bladder predominantly with eosinophils, associated with fibrosis with or without muscle necrosis
Source: Characterized by transmural inflammation of the bladder predominantly with eosinophils, associated with fibrosis with or without muscle necrosis
Parent: https://www.ncbi.nlm.nih.gov/pubmed/15857336
Disease association: Eosinophilia

**Comment by pnrobinson** [comment]
- [ ] I think that Eosinophilic cystitis is the primary label for the disease entity, but it also describes the main phenotypic feature of the disease.
- [ ] Therefore, I would suggest the term label Eosinophilic bladder infiltration. 
- Should we state that the infiltration is typically transmural? (see e.g. https://www.ncbi.nlm.nih.gov/pubmed/28829677).
 - [ ] Proposed parent Urinary bladder inflammation (https://hpo.jax.org/app/browse/term/HP:0100577)

**Comment by nicolevasilevsky** [comment]
Eosinophilic cystitis is a synonym for MONDO_0023073
